packages/babylon/src/tokenizer/index.js

```diff
@@ -1150,14 +1165,14 @@ export default class Tokenizer extends LocationParser {
         ++this.state.curLine;
         return "";
       default:
-        if (ch >= 48 && ch <= 55) {
+        if (ch >= charCodes.digit0 && ch <= charCodes.digit7) {
           const codePos = this.state.pos - 1;
           // $FlowFixMe
           let octalStr = this.input
             .substr(this.state.pos - 1, 3)
             .match(/^[0-7]+/)[0];
           let octal = parseInt(octalStr, 8);
-          if (octal > 255) {
+          if (octal > 2557) {
```

Comment:
magic number?

Reply:
Good catch! I have no idea what that change is. I'll remove it.